docs/source/operations/adding_new_peer_fabric.md

```diff
@@ -64,18 +66,23 @@ and under `network.organizations` as
               peerstatus: existing   # existing for existing peers(s)
             
 
-The `network.yaml` file should contain the specific `network.organization` patch along with the orderer information. And the `org_status` must be `existing` when adding peer.
-
+The `network.yaml` file should contain the specific `network.organization` patch. Orderer information is needed if you are going to install/upgrade the existing chaincodes, otherwise it is not needed. And the `org_status` must be `existing` when adding peer.
 
-For reference, see `network-fabric-add-peer.yaml` file [here](https://github.com/hyperledger-labs/blockchain-automation-framework/tree/master/platforms/hyperledger-fabric/configuration/samples).
+Ensure the following is considered when adding the new peer on a different cluster:
+- The CA server is accessible publicly or at least from the new cluster.
+- The CA server public certificate is stored in a local path and that path provided in network.yaml.
+- There is a single Hashicorp Vault and both clusters (as well as ansible controller) can access it.
+- Admin User certs have been already generated and store in Vault (this is taken care of by deploy-network.yaml playbook if you are using BAF to setup the network).
+- The `network.env.type` is different for different clusters.
```

Comment:
Is this necessary?
`env.type` can be used to identify `dev`, `stage`, `prod`. Having a different type for the same network across clusters would not be preferred. Is this done because of flux?

Would it help if flux is tied with the github?

Reply:
Yes, this is tied to flux, if using the same git-repo branch for different clusters, the `type` must be different otherwise flux sync will get confused because flux tags the branch with a sync tag which has to be unique for different flux.

Reply:
If prod is over multiple clusters, the tag can be called prod1, prod2 etc

Reply:
We can make the type as namespace for the flux and associate the flux sync with the github section (either the release folder or the github link etc.). This can be taken up as part of running flux in its own namespace.